package.json

```diff
@@ -47,7 +47,7 @@
     "lint:scss": "stylelint {demosplan/DemosPlanCoreBundle/Resources/client/scss,projects}/**/*.scss"
   },
   "dependencies": {
-    "@demos-europe/demosplan-ui": "^0.3.5",
+    "@demos-europe/demosplan-ui": "github:demos-europe/demosplan-ui#f_T34290_add_checkResponse_to_dorequest",
```

Comment:
this change is not needed when working with the docker service. just check out the appropriate branch in the other repo.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2538/commits/ac514d86ab4a70dead6bdb9edd85064fe0be3dc6